btr-api/migrations/versions/20240814_1520_877c0b71efb6_.py

```diff
@@ -0,0 +1,38 @@
+"""empty message
```

Comment:
can remove this migration for 'previous_payload' I think?

Reply:
Done